superset/reports/schemas.py

```diff
@@ -178,6 +179,7 @@ class ReportSchedulePostSchema(Schema):
     )
 
     recipients = fields.List(fields.Nested(ReportRecipientSchema))
+    report_format = fields.String(default=ReportDataFormat.VISUALIZATION)
```

Comment:
Let's add a validation here also:

```
validate=validate.OneOf(choices=tuple(key.value for key in ReportDataFormat)),
```
This will enrich the OpenAPI spec also